helpers/lib/common.js

```diff
@@ -24,10 +24,20 @@ function isTruthy(val) {
     }
 }
 
+function matchesPredicate(element, predicate, predicateKeys) {
+    for (let j = 0; j < predicateKeys.length; j++) {
```

Comment:
Now that this is not nested, probably better to use `i` for your loop variable

Reply:
ok.